.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -5,7 +5,7 @@ about: Create a report to help us improve
 
 Be sure to check the existing issues (both open and closed!), and make sure you are running the latest version of Pipenv.
 
-Check the [diagnose documentation](https://docs.pipenv.org/diagnose/) for common issues before posting! We may close your issue if it is very similar to one of them. Please be considerate, or be on your way.
+Check the [diagnose documentation](/docs/diagnose.rst) for common issues before posting! We may close your issue if it is very similar to one of them. Please be considerate, or be on your way.
```

Comment:
So the documentation is actually available at https://docs.pipenv.org but there is nothing available at https://docs.pipenv.org/docs/diagnose/ because the subfolder `docs` doesn't exist.  The actual diagnose documentation is at https://docs.pipenv.org/en/latest/diagnose/

Reply:
Thanks for the review. I updated Diagnose URL as you mentioned. (https://docs.pipenv.org/en/latest/diagnose/)